core/controllers/reader.py

```diff
@@ -21,7 +21,7 @@
 import jinja2
 
 from core.controllers import base
-from core.domain.classifier import lda_string_classifier
+from core.domain.classifier import LDAStringClassifier
```

Comment:
@seanlip I haven't really touched reader.py other than updating the module name. Actually, reader.py has to be updated to use the classifier_registry. I have left these changes for the next phase (since I've planned to revamp the classification specific functions in reader.py anyway).

For now, I have only updated the names so that the things do not break.